### DISCUSSION TITLE
Help with HTML integration

### DISCUSSION THREAD

**Discussion by Rohlik** [matteobruni/tsparticles#1656] [Q&A] [answered]
Hello 👋🏼 ,
I have a problem and I probably doing something wrong as my simple HTML site is not working :-/
I have something like this in `index.html`:
```
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  <head>
  <meta http-equiv="content-type" content="text/html; charset=utf-8">
  <meta name="generator" content="PSPad editor, www.pspad.com">
  <title></title>
  </head>
  <body>
<div id="tsparticles">Hello World</div>
<script src="https://cdnjs.cloudflare.com/ajax/libs/tsparticles/1.28.2/tsparticles.min.js" integrity="sha512-XJdtD5bxGbJ1VVjp17f/fPkoM4STYtE0x6DNeRp1nxJn5l7faPXLMhaVXnTIwOeqQin/+CdzPQ0bbVaqgpdfeA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
  </body>
</html>
```
and in same directory I have also `app.js` with default content mentioned here https://github.com/matteobruni/tsparticles/blob/main/README.md#usage.  
Only thing what i changed was path to my json config:
`.loadJSON("tsparticles", "particlesjs-config.json")`

When I open that page I see just my `Hello World` and no effects at all 😞 .
What I did wrong? What piece is missing in my puzzle?

**Comment by matteobruni** [comment, marked as answer]
Hello,

Can you share the contents of `"particlesjs-config.json"`?

Anyway I see two anomalies in your code:

``` html
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>

<head>
    <meta http-equiv="content-type" content="text/html; charset=utf-8">
    <meta name="generator" content="PSPad editor, www.pspad.com">
    <title></title>
</head>

<body>
    <!-- The tsParticles div should be empty -->
    <div id="tsparticles"></div>

    <script src="https://cdnjs.cloudflare.com/ajax/libs/tsparticles/1.28.2/tsparticles.min.js"
        integrity="sha512-XJdtD5bxGbJ1VVjp17f/fPkoM4STYtE0x6DNeRp1nxJn5l7faPXLMhaVXnTIwOeqQin/+CdzPQ0bbVaqgpdfeA=="
        crossorigin="anonymous" referrerpolicy="no-referrer"></script>

    <!-- You missed the app.js in the page, this must be included after the tsParticles one -->
    <script src="app.js"></script>
</body>

</html>
```

I added comments and fixes in the code

**Reply by Rohlik** [reply]
Thank you @matteobruni for your time.
I edited my files according to your feedback and for better debugging I uploaded to my test page https://simi.rohlik.xyz
I would like to use default `confetti` config - https://simi.rohlik.xyz/assets/particlesjs.json
Sadly, it still doesn't work. I tried in Firefox ESR and Chrome.

**Reply by matteobruni** [reply]
If you want to use the `"confetti"` shape you must include the additional script since that shape is not included in basic tsParticles.

You can find the scripts here: https://www.jsdelivr.com/package/npm/tsparticles-shape-confetti

The confetti shape script must be added after the tsParticles one and before your `app.js`

**Reply by Rohlik** [reply]
@matteobruni You make my day 👍🏼. It works now. Thank you.
